pymedphys/_version.py

```diff
@@ -1,4 +1,5 @@
-# pylint: disable=invalid-name, missing-docstring
+# DO NOT EDIT
+# This file is autogenerated by `pymedphys dev propagate`
 
 version_info = [0, 34, 0, "dev2"]
-__version__ = "0.34.0dev2"
+__version__ = "0.34.0.dev2"
```

Comment:
is this proper semver ?  I thought that a dash in the 3rd dot-field for indicating pre-release


```
Precedence for two pre-release versions with the same major, minor, and patch version MUST be determined by comparing each dot separated identifier from left to right until a difference is found as follows:

Identifiers consisting of only digits are compared numerically.

Identifiers with letters or hyphens are compared lexically in ASCII sort order.

Numeric identifiers always have lower precedence than non-numeric identifiers.

A larger set of pre-release fields has a higher precedence than a smaller set, if all of the preceding identifiers are equal.

Example: 1.0.0-alpha < 1.0.0-alpha.1 < 1.0.0-alpha.beta < 1.0.0-beta < 1.0.0-beta.2 < 1.0.0-beta.11 < 1.0.0-rc.1 < 1.0.0.
```

Reply:
Yup, fair call. Here's an example in the wild:

https://github.com/python-semver/python-semver/blob/ddf2b304ebc0bf4a8083a13372c908c8dc6b8dff/semver.py#L28